guides/source/asset_pipeline.md

```diff
@@ -1195,3 +1041,51 @@ it as a preprocessor for your MIME type.
 ```ruby
 Sprockets.register_preprocessor 'text/css', AddComment
 ```
+
+
+Alternative Libraries
+------------------------------------------
+
+Over the years there have been multiple default approaches for handling the assets. The web evolved and we started to see more and more Javascript-heavy applications. In The Rails Doctrine we believe that [The Menu Is Omakase](https://rubyonrails.org/doctrine#omakase) so we focused on the default setup: **Sprockets with Import Maps**.
+
+We are aware that there are no one-size-fits-it-all solutions for the various JavaScript and CSS frameworks/extensions available. There are other bundling libraries in the Rails ecosystem that should empower you in the cases where the default setup isn't enough.
+
+### [jsbundling-rails](https://github.com/rails/jsbundling-rails)
+
+A Node.js dependent alternative to the `importmap-rails` way of bundling JavaScript with [esbuild](https://esbuild.github.io/), [rollup.js](https://rollupjs.org/), or [Webpack](https://webpack.js.org/).
+
+The gem provides `yarn build --watch` process to automatically generate output in development. For production, it automatically hooks `javascript:build` task into `assets:precompile` task to ensure that all your package dependencies have been installed and JavaScript has been built for all entry points.
+
+**When to use instead of `importmap-rails`?** If your JavaScript code depends on transpiling so if you are using [Babel](https://babeljs.io/), [TypeScript](https://www.typescriptlang.org/) or React `JSX` format then `jsbundling-rails` is the correct way to go.
+
+### [Webpacker/Shakapacker](https://guides.rubyonrails.org/webpacker.html)
+
+Webpacker has been the default JavaScript pre-processor and bundler for Rails 5 and 6. `Webpacker` successor is now [`shakapacker`](https://github.com/shakacode/shakapacker) and it's advised to use it.
```

Comment:
```suggestion
Webpacker was the default JavaScript pre-processor and bundler for Rails 5 and 6. It has now been retired. A successor called [`shakapacker`](https://github.com/shakacode/shakapacker) exists, but is not maintained by the Rails team or project.
```

Reply:
The first line of README in Shakapacker repo goes like this:
> Official, actively maintained successor to [rails/webpacker](https://github.com/rails/webpacker).

If it's not maintained by Rails team then maybe they shouldn't use the `official` word, but anyway I like the rewording there and going to commit it.

Reply:
I'm not sure what official means in that context. It was added in https://github.com/shakacode/shakapacker/pull/14

I suggested that wording because it's not part of the `rails/` github org so isn't owned by the same core team.

cc @justin808 you may have some thoughts on this (or on this PR in general).

Reply:
It's official in the sense that @dhh recommends it as the successor, and he came up with the name "shakapacker."

See https://github.com/rails/webpacker.